tests/tests_unit.c

```diff
@@ -748,7 +748,6 @@ static void test_mnemonic(void)
         u_assert_str_eq(m, *b);
         wallet_mnemonic_to_seed(m, "TREZOR", seed, 0);
         u_assert_mem_eq(seed, utils_hex_to_uint8(*c), strlen(*c) / 2);
-        u_assert_str_eq(wallet_mnemonic_from_index(wallet_index_from_mnemonic(*b)), *b);
```

Comment:
Can we add a test for the new xpub export if technically possible?
I think we should try to keep test coverage as hight as possible.